### PR TITLE
ATR-3 crawls data from URL, parses and sends email every minute whenever ATR Index gets greater than 2000

### PR DESCRIPTION
**Pull request by jawur**
[JIRA Status](https://netsells.atlassian.net/browse/ATR-3)

## Description

*Added artisan command, schedule, service/RaceMonitor class, config, 2 variables in env file and removed RaceController with routes. Runs background*

## Links
Related links below:

* https://netsells.atlassian.net/browse/ATR-3

### PR requires review from both frontend and backend
No

@netsells/lead-developers - please review

### CODE REVIEW

File: app/Console/Commands/CrawlData.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+     * @var string
+     */
+    protected $signature = 'email:send';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Email the top 10 table Race and ATR Index data, email:send {email?}';
```
Comment (craigsansam) [inline app/Console/Commands/CrawlData.php:24]: Putting the signature in the description is a nice touch, but isn't required. If you run `php artisan email:send --help` the output will display the parameters etc

Comment (jawur) [reply app/Console/Commands/CrawlData.php:24]: @craigsansam, understood! Do you want me to remove it?

Comment (AdamHutchison) [reply app/Console/Commands/CrawlData.php:24]: No you can leave it in for now Kirill, was just an FYI from Craig

File: services/RaceMonitor/RaceMonitor.php [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+        $this->validateEmail();
+
+        $dom = $this->getDOM($this->target);
+        $data = $this->getData($dom);
+        $this->sendEmail($data);
+
+        return $this->getMessage();
+    }
+
+    //------------------------------------------------------------------------------------------------------------------
```
Comment (AdamHutchison) [inline services/RaceMonitor/RaceMonitor.php]: Can you remove this line comment please Kirill? We try not to clutter the class with anything that isn't essential

File: services/RaceMonitor/RaceMonitor.php [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+    /**
+     * Returns message
+     *
+     * @return string
+     */
+    public function getMessage()
+    {
+        return $this->message;
+    }
```
Comment (AdamHutchison) [inline services/RaceMonitor/RaceMonitor.php:160]: It's best to group all private methods together so can you move this public method tobe with the others please?

**Review by AdamHutchison — CHANGES_REQUESTED** [review]

File: services/RaceMonitor/RaceMonitor.php (hunk 2 above)
Comment (jawur) [reply services/RaceMonitor/RaceMonitor.php]: @AdamHutchison done :+1:

File: services/RaceMonitor/RaceMonitor.php (hunk 3 above)
Comment (jawur) [reply services/RaceMonitor/RaceMonitor.php:160]: @AdamHutchison done :+1:

**Pull request merged**